### PR TITLE
LG-3043: map bridge certs

### PR DESCRIPTION
**Pull request by solipet**
Significantly reduces the number of root CA's required and ensures that we don't end up in an infinite loop when building up the signatory chain.

### CODE REVIEW

File: app/services/policy_mapping_service.rb [hunk 1]
```diff
@@ -18,6 +18,7 @@ def chain(set = [])
     @chain ||= begin
       signer = store[certificate.signing_key_id]
       while signer
+        break if set.include? signer
```
[the remark this answers is not in the document]
Comment (jmhooper) [reply app/services/policy_mapping_service.rb]: Our `Certificate` operator actually overrides `==`. It should be safe to compare 2 `Certificate` objects, but not 2 `OpenSSL::X509::Certificate` objects.

Ref: https://github.com/18F/identity-pki/blob/master/app/models/certificate.rb#L37-L41

File: config/application.yml.example [hunk 2]
```diff
@@ -11,14 +11,20 @@ aws_http_timeout: '5'
...
-  68:84:15:48:8C:54:70:7F:2D:12:58:0E:EC:1C:78:EF:3C:2E:59:64,\
-  6C:8A:94:A2:77:B1:80:72:1D:81:7A:16:AA:F2:DC:CE:66:EE:45:C0,\
-  BD:C1:B9:6B:4D:F4:1D:EC:30:90:BF:62:73:C0:84:33:F2:71:24:85,\
-  6F:83:FE:82:50:64:65:77:3E:FD:DF:03:9A:CE:29:D1:2F:30:CC:EC"
+  BD:C1:B9:6B:4D:F4:1D:EC:30:90:BF:62:73:C0:84:33:F2:71:24:85"
+
+# DoD root identifiers:
+# 49:74:BB:0C:5E:BA:7A:FE:02:54:EF:7B:A0:C6:95:C6:09:80:70:96 - DoD Root CA 2
+# 6C:8A:94:A2:77:B1:80:72:1D:81:7A:16:AA:F2:DC:CE:66:EE:45:C0 - DoD Root CA 3
+# BD:C1:B9:6B:4D:F4:1D:EC:30:90:BF:62:73:C0:84:33:F2:71:24:85 - DoD Root CA 5
```
Comment (solipet) [inline config/application.yml.example:27]: Isn't this `DoD Root CA 3`?

Comment (jmhooper) [reply config/application.yml.example]: I put these in here to be helpful and then I got them all wrong. I just pushed a commit to fix this comment.

File: app/services/policy_mapping_service.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (zachmargolis) [reply app/services/policy_mapping_service.rb:22]: oh perfect, thanks for that!

**Pull request merged**